app/models/foreman_virt_who_configure/config.rb

```diff
@@ -1,5 +1,5 @@
 module ForemanVirtWhoConfigure
-  class Config < ActiveRecord::Base
+  class Config < ApplicationRecord
```

Comment:
What does `ApplicationRecord` do that `ActiveRecord::Base` can not do?

Reply:
rubocop: `Rails/ApplicationRecord: Models should subclass ApplicationRecord.`

Reply:
Subclass `ApplicationRecord` is a better approach than `ActiveRecord::Base`